lib/vmpooler/api/v1.rb

```diff
@@ -68,9 +68,9 @@ def fetch_single_vm(template)
       end
 
       template_backends.each do |template_backend|
-        vm = backend.spop("vmpooler__ready__#{template_backend}")
+        vm = backend.smembers("vmpooler__ready__#{template_backend}")[-1]
```

Comment:
is `smember()` nil-safe?

Reply:
Just tested, and it looks like it should be alright. When smembers is empty it returns an empty array, and requesting the last element of that empty array produces nil, so I think it should be ok.